spec/features/events.js

```diff
@@ -0,0 +1,31 @@
+import webdriverio from 'webdriverio';
+import chai from 'chai';
+import { config } from '../wdio.config';
+
+// @TODO FIX ME! These tests work on my local machine, but not in Circle CI :(
```

Comment:
Viewing events would be useful, but adding events possibly not so useful if we're planning on moving away from managing events/content in this project.

We could come back to this once we no longer using CouchDB.